tests/h/views/api/users_test.py

```diff
@@ -7,105 +7,19 @@
-                                   view):
-        basic_auth_creds.return_value = ('C69BA868-5089-4EE4-ABB6-63A1C38C395B',
-                                         'somerandomsecret')
-        pyramid_request.json_body = self.valid_payload(type_)
-
-        with pytest.raises(ClientUnauthorized):
-            view(pyramid_request)
-
-    def test_raises_when_client_secret_invalid(self,
-                                               auth_client,
-                                               basic_auth_creds,
-                                               pyramid_request,
-                                               type_,
-                                               view):
-        basic_auth_creds.return_value = (auth_client.id, 'incorrectsecret')
-        pyramid_request.json_body = self.valid_payload(type_)
-
-        with pytest.raises(ClientUnauthorized):
-            view(pyramid_request)
-
-    def test_raises_for_public_client(self,
-                                      factories,
-                                      basic_auth_creds,
-                                      pyramid_request,
-                                      type_,
-                                      view):
-        auth_client = factories.AuthClient(authority='weylandindustries.com')
-        basic_auth_creds.return_value = (auth_client.id, '')
-        pyramid_request.json_body = self.valid_payload(type_)
-
-        with pytest.raises(ClientUnauthorized):
-            view(pyramid_request)
-
-    def test_raises_for_invalid_client_grant_type(self,
-                                                  factories,
-                                                  basic_auth_creds,
-                                                  pyramid_request,
-                                                  type_,
-                                                  view):
-        auth_client = factories.ConfidentialAuthClient(authority='weylandindustries.com',
-                                                       grant_type=GrantType.authorization_code)
-        basic_auth_creds.return_value = (auth_client.id, auth_client.secret)
-        pyramid_request.json_body = self.valid_payload(type_)
-
-        with pytest.raises(ClientUnauthorized):
-            view(pyramid_request)
-
-    def valid_payload(self, type_):
-        payload = {'email': 'jeremy@weylandtech.com',
-                   'display_name': 'Jeremy Weyland'}
-
-        if type_ == 'create':
-            payload.update({'authority': 'weylandindustries',
-                            'username': 'jeremy'})
-
-        return payload
-
-
 @pytest.mark.usefixtures('auth_client',
-                         'basic_auth_creds',
+                         'request_auth_client',
                          'user_signup_service',
                          'user_unique_svc')
 class TestCreate(object):
-    @pytest.mark.usefixtures('valid_auth')
     def test_signs_up_user(self,
                            factories,
```

Comment:
Plz remove the `factories` param here as it's not used (trying to cut down on unused fixtures).